shared/bazel/rules/BUILD.bazel

```diff
@@ -19,3 +20,110 @@ pkg_files(
     prefix = "META-INF",
     visibility = ["//visibility:public"],
 )
+
+config_setting(
```

Comment:
I'd prefer this gets merged up with https://github.com/wpilibsuite/rules_bzlmodrio_toolchains/blob/main/conditions/BUILD.bazel

Reply:
Sounds good, I went with expedient to learn, now I get to move the pieces to the right spot.  I'll start splitting things out into more reviews.